### ISSUE TITLE
TTL of 10s for optimistic caching takes websites often longer to load

### ISSUE THREAD

**Issue by JsBergbau** [AdguardTeam/AdGuardHome#6094]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Linux, ARM64

### Installation

GitHub releases or script from README

### Setup

On one machine

### AdGuard Home version

v0.107.36

### Action

```sh
nslookup -debug -type=a 'www.example.com' '$YOUR_AGH_ADDRESS'
```

Do a dig for a previously looked up DNS entry, where the TTL has passed. With optimistic caching you will now get the cached result with a TTL of 10 seconds.

It seems a lot more pages change their IPs more often than expected. When the page doesn't load I have now to wait 10 seconds until a retry is successful because only after that period a new DNS request with the resulting correct IP is made (combination of windows resolver and Fritzbox, which is setup to use AdGuard as upstream resolver).

### Expected result

Optimistic caching is done to speed up webbrowsing. When you have to wait at least 10 seconds until wegpages are correctly loaded then this feature does the opposite. 
So I propose to set a TTL of 1s for optimistic results. This will hurt no one but speed up page loading in cases where the IP changed. 
This is also only a very minor change of code and can be done very fast.

### Actual result

TTL for optimistic caching is 10 seconds, which leads to problems, see above.

**Comment by ainar-g** [comment]
Duplicate of #5621
